packages/fiori/test/specs/Wizard.spec.js

```diff
@@ -53,6 +55,11 @@ describe("Wizard general interaction", () => {
 			"First step in the header has aria-label.");
 		assert.strictEqual(await step2InHeaderRoot.getAttribute("aria-label"), step2Text,
 			"Second step (inactive) in the header has aria-label.");
+
+		assert.strictEqual(await wizStep.getAttribute("role"), "region",
+			"Step has role 'region'");
+		assert.strictEqual(await wizStep.getAttribute("aria-label"), await step1InHeader.getProperty("title"),
+			"Step has aria-label set to its title.");
```

Comment:
From my side, this assert does not pass:
**Step has aria-label set to its title.: expected '1 Product type' to equal ''**
Could you please check?

Reply:
As I can see, the build is failing with the same error